File: QuarkusLokiLoggerDeployment/src/main/java/net/codinux/log/loki/quarkus/QuarkusLokiLoggerProcessor.java
package net.codinux.log.loki.quarkus;

import io.quarkus.deployment.annotations.BuildStep;
import io.quarkus.deployment.builditem.FeatureBuildItem;

public class QuarkusLokiLoggerProcessor {

    private static final String FEATURE = "loki-logger";

    @BuildStep
    public FeatureBuildItem feature() {
        return new FeatureBuildItem(FEATURE);
    }
}

File: QuarkusLokiLoggerDeployment/src/main/kotlin/net/codinux/log/loki/quarkus/QuarkusLokiLoggerProcessor.kt
package net.codinux.log.loki.quarkus

import io.quarkus.deployment.annotations.BuildStep
import io.quarkus.deployment.builditem.FeatureBuildItem

class QuarkusLokiLoggerProcessor {

    companion object {
        private const val Feature = "loki-logger"
    }


    @BuildStep
    fun feature() = FeatureBuildItem(Feature)

}